docs/glossary.md

```diff
@@ -24,7 +24,7 @@ orthography, make some common spelling mistakes, and may be lacking
 accents or aspiration marks where they would be required in
```

Comment:
accents -> diacritics
aspiration is marked with the character -h-, not a diacritic, in any Cree orthography (in comparison to scifi-esque use of an apostrophe in e.g. B'elanna.
In SRO, one can use the apostrophe to indicate a reduced/deleted short /i/, i.e. tân'si for tânisi.